service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/ServiceLocator.scala

```diff
@@ -39,6 +47,16 @@ trait ServiceLocator {
   def locate(name: String, serviceCall: Descriptor.Call[_, _]): Future[Option[URI]]
```

Comment:
I wonder if we should even deprecate the single-value `locate` methods. What do you think?